src/main/java/Parser.java

```diff
@@ -0,0 +1,95 @@
+public class Parser {
+    private String[] inputs;
+    private String command;
+    private String description;
+    private String timing;
+
+
+    /**
+     * Constructor for Parser Class.
+     *
+     * @param input User input.
+     */
+    public Parser(String input) {
+        this.inputs = input.split(" ");
+        this.command = inputs[0];
```

Comment:
Maybe you can create a Command class ad subclasses for all the different types of commands, so it will have better abstraction